bigchaindb/backend/mongodb/changefeed.py

```diff
@@ -104,3 +75,37 @@ def get_changefeed(connection, table, operation, *, prefeed=None):
 
     return MongoDBChangeFeed(table, operation, prefeed=prefeed,
                              connection=connection)
+
+
+_FEED_STOP = False
+"""If it's True then the changefeed will return when there are no more items.
+"""
+
+
+def run_changefeed(conn, table, last_ts):
+    """Encapsulate operational logic of tailing changefeed from MongoDB
+    """
+    while True:
+        try:
+            # XXX: hack to force reconnection, in case the connection
+            # is lost while waiting on the cursor. See #1154.
+            conn._conn = None
+            namespace = conn.dbname + '.' + table
+            query = conn.query().local.oplog.rs.find(
+                {'ns': namespace, 'ts': {'$gt': last_ts}},
+                {'o._id': False},
+                cursor_type=pymongo.CursorType.TAILABLE_AWAIT
+            )
+            cursor = conn.run(query)
+            logging.debug('Tailing oplog at %s/%s', namespace, last_ts)
+            while cursor.alive:
+                try:
+                    record = cursor.next()
+                    yield record
+                    last_ts = record['ts']
+                except StopIteration:
+                    if _FEED_STOP:
+                        return
```

Comment:
Since this is being used as an iterator shouldn't we raise a `StopIteration` at this point?
Whatever is consuming this may not be expecting it to return `None`